README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
+- check that your internet connection can reach `clients.IMCO_DOMAIN`
+- make sure that your firewall lets you access to <https://clients.IMCO_DOMAIN:886>
+- check that `client.xml` is pointing to the correct certificates location
+- if `concerto` executes but only shows server commands, you are probably trying to use `concerto` from a commissioned server, and the configuration is being read from `/etc/imco`. If that's the case, you should leave `concerto` configuration untouched so that server commands are available for our remote management.
+
+## Usage
 
-# Usage
 We include the most common use cases here. If you feel there is a missing a use case here, open an issue or contact us at <enquiries@concerto.io>.
 
 ## Wizard
-The Wizard command for Concerto CLI is the command line version of our `Quick add server` in the Concerto Web UI.
 
-<img src="./docs/images/webwizard.png" alt="web wizard" width="500px" >
+The Wizard command for IMCO CLI is the command line version of our `Quick add server` in the IMCO's Web UI.
+
+<img src="./docs/images/webwizard.png" alt="Web Wizard" width="500px" >
 
 Wizard is the quickest way to install a well known stack in a cloud server. You can get an idea of what the wizard does using the command `concerto wizard` without further subcommands:
-```
+
+```bash
 $ concerto wizard
 NAME:
-   concerto wizard - Manages wizard related commands for apps, locations, cloud providers, server plans
+    - Manages wizard related commands for apps, locations, cloud providers, server plans
+
 USAGE:
-   concerto wizard command [command options] [arguments...]
+    command [command options] [arguments...]
+
 COMMANDS:
-   apps         Provides information about apps
-   cloud_providers  Provides information about cloud providers
-   locations        Provides information about locations
-   server_plans     Provides information about server plans
-   help, h      Shows a list of commands or help for one command
-OPTIONS:
-   --help, -h   show help
+     apps             Provides information about apps
+     cloud_providers  Provides information about cloud providers
+     locations        Provides information about locations
+     server_plans     Provides information about server plans
+...
 ```
 
-Concerto Wizard lets you select the application layer, the location, the cloud provider for that location, and finally the hostname. Concerto Wizard takes care of the details.
-
+IMCO CLI Wizard lets you select the application layer, the location, the cloud provider account for that location, and finally the hostname. IMCO CLI Wizard takes care of the details.
 
-If you haven't configured you cloud provider account yet, you can do it from the Concerto Web UI, or using `concerto settings cloud_accounts` commands
+If you haven't configured your cloud provider accounts yet, you can do it from the IMCO's Web UI, or using `concerto settings cloud_accounts` commands
```

Comment:
Cloud account management is no longer possible from IMCO (Web or API) interface, only the listing is available. Cloud accounts are managed as cloud subscriptions from APS (or any other platform managing it).